src/components/CadenceVersion.tsx

```diff
@@ -0,0 +1,22 @@
+import React, { useState, useEffect } from 'react';
+
+const API = process.env.PLAYGROUND_API;
+
+export const Version = () => {
+  const [version, setVersion] = useState('--');
+
+  useEffect(() => {
+    getCadenceVerion();
+  }, []);
+
+  const url = `${API}/utils/version`;
```

Comment:
When I look at this code, I feel like we could add this information to the "app state". But it's not an issue rn since we're not using multiple instance of this component.

Reply:
Yeah, I started with idea that we can put it somewhere inside `useProject` hook and then discarded it for exact same reason you've mentioned 😉